src/FixedDataTable.js

```diff
@@ -346,7 +351,7 @@ var FixedDataTable = React.createClass({
   },
 
   _shouldHandleWheelX(/*number*/ delta) /*boolean*/ {
-    if (this.props.overflowX === 'hidden') {
+    if (this.props.overflowX === 'hidden' || this.props.disableWheelHandler) {
```

Comment:
I'm not really sure (we inherited this code from the original repo)
I'm guessing one reason is that it doesn't have access to this.props yet.

@wcjordan This seems like a big memory leak, creating unnecessary wheel handlers if the table switches from overflow on and off.

It seems unnecessary to create it in that function, since the handler is already initialized in componentWillMount.